modules/appeals_api/config/schemas/v2/200995.json

```diff
@@ -123,25 +123,15 @@
                     }
                   },
                   "required": ["evidenceType"],
-                  "anyOf": [
-                    {
-                      "not": {
-                        "properties": {
-                          "evidenceType": { "contains": { "enum": ["retrieval"] } }
-                        }
-                      }
-                    },
-                    { "required": ["retrieveFrom"] }
-                  ]
-                },
-                "noticeAcknowledgement": {
-                  "enum": [true, false]
+                  "if": { "properties": { "evidenceType": { "const": "retrieval" } } },
+                  "then": { "required": ["retrieveFrom"] }
                 },
+                "noticeAcknowledgement": { "type": "string", "enum": ["yes", "no"] },
                 "socOptIn": { "type": "boolean" }
               },
               "required": ["veteran", "evidenceSubmission", "socOptIn", "noticeAcknowledgement"],
               "if": { "properties": { "benefitType": { "const": "compensation" } } },
-              "then": { "properties": { "noticeAcknowledgement": { "const": true } } }
+              "then": { "properties": { "noticeAcknowledgement": { "const": "yes" } } }
```

Comment:
Does the const stuff fail when it's set to `false` after you update the acknowledgement to type boolean?

Reply:
yes, I believe that was the problem and part of the reason I changed it back after trying to set it to a boolean. I'll revisit.